flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/service/FlinkService.java

```diff
@@ -234,6 +238,22 @@ private JarUploadResponseBody jarUpload(FlinkSessionJob sessionJob, Configuratio
         }
     }
 
+    private void jarDelete(Configuration conf, String jarId) {
```

Comment:
nit: could we please change the name to `deleteJar`?

Reply:
Fixed